packages/home-spa/src/service/service.js

```diff
@@ -1,6 +1,8 @@
-const listHomeType = `
-query {
-  listHomeType {
+const getHomeTypeBySPA = `
+query HomePageQuery {
+  getHomeTypeBy(input: {
+    entityType: "spa"
+  }) {
```

Comment:
Home microservice is nearing deprication. It will be great if you can include the app service queries

Reply:
Is the app service available ?